interop/jvm/src/main/scala/scalaz/zio/interop/catz.scala

```diff
@@ -1,21 +1,27 @@
 package scalaz.zio
 package interop
 
+import scala.util.control.NonFatal
+
 import cats.effect
 import cats.effect.Effect
 import cats.syntax.all._
 
-import scala.util.control.NonFatal
-
 object catz extends RTS {
 
-  implicit val catsEffectInstance: Effect[Task] = new Effect[Task] {
+  case class FromToThrowable[E](to: E => Throwable, from: Throwable => E)
```

Comment:
You can do better:

```scala
case class FromToThrowable[E](to: E => Throwable, from: PartialFunction[Throwable, E])
```

If `from` is not defined for some `t: Throwable`, then you call `IO.terminate(t)`. 

Of course, a user can always do this themselves, but it's common enough it may warrant baking in.

Reply:
I think i'll go with `from: E => Option[Throwable]`, let's stick to only one anti-pattern per PR :laughing:

Reply:
Done...